geoips/plugins/modules/readers/ssmis_binary.py

```diff
@@ -140,30 +140,31 @@ def read_ssmis_data_file(fname, metadata_only=False):
     f1 = open(fname, "rb")
 
     # READ HEARDER
-    sw_rev = np.fromstring(f1.read(2), dtype=np.dtype("short")).byteswap()[0]
-    endian, fileid = np.fromstring(f1.read(2), dtype=np.dtype("int8")).byteswap()
+    # sw_rev = np.fromstring(f1.read(2), dtype=np.dtype("short")).byteswap()[0]
+    # endian, fileid = np.fromstring(f1.read(2), dtype=np.dtype("int8")).byteswap()
     rev = np.fromstring(f1.read(4), dtype=np.dtype("int32")).byteswap()
     year = np.fromstring(f1.read(4), dtype=np.dtype("int32")).byteswap()
-    jday = np.fromstring(f1.read(2), dtype=np.dtype("short")).byteswap()
-    hour, minu = np.fromstring(f1.read(2), dtype=np.dtype("int8")).byteswap()
+    # jday = np.fromstring(f1.read(2), dtype=np.dtype("short")).byteswap()
+    # hour, minu = np.fromstring(f1.read(2), dtype=np.dtype("int8")).byteswap()
     satid, nsdr = np.fromstring(f1.read(4), dtype=np.dtype("short")).byteswap()
-    spare1, spare2, spare3 = np.fromstring(
-        f1.read(3), dtype=np.dtype("int8")
-    ).byteswap()
-    proc_stat_flags = np.fromstring(f1.read(1), dtype=np.dtype("int8")).byteswap()
-    spare4 = np.fromstring(f1.read(4), dtype=np.dtype("int32")).byteswap()
+    # spare1, spare2, spare3 = np.fromstring(
```

Comment:
Oooh, this won't work - it needs to read out of the file, I think.  Even if these aren't used, it will get out of sync if those reads aren't there.  Maybe just don't have it actually set the variables for the ones that aren't used.

Reply:
We should just have it set them to _. Either that or is there a way to just seek forward?

Reply:
Just added the # NOQA comment to the affected lines. Explained why at the top of the file.